docs/_lab/09-writing-advanced-plans/index.md

```diff
@@ -48,7 +48,7 @@ Functions used in this example:  [run_task](https://puppet.com/docs/bolt/1.x/pla
 Run the plan.
 
 ```bash
-bolt plan run exercise9::yesorno nodes=all --modulepath ./modules
+bolt plan run exercise9::yesorno nodes=linux
```

Comment:
I think this should go back to all? As far as I know they should all be all?

Reply:
exercise9::yesorno fails on Windows because of a task file error. The `.py` extension isn't enabled on that node.